components/avatar.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Decode UI</title>
+    <link href="/style.css" rel="stylesheet">
+    <link rel="shortcut icon" href="/decodeui-favicon.ico" type="image/x-icon">
+</head>
+
+<body>
+    <nav>
+        <a href="/index.html">
+            <img src="/DECODE UI.png" class="hero"></a>
+        <button class="button-primary button-nav padding-1">
+            <a href="https://github.com/pragyasabharwal/decode-ui" id="github-link">View on GitHub</a>
+        </button>
+    </nav>
+    <div class="main-grid">
+        <div class="side-nav padding-10 margin-3">
+            <a href="/documentation.html">
+                <li class="padding-7">Installation</li>
+            </a>
+            <a href="avatar.html">
+                <li class="padding-7 active">Avatar</li>
+            </a>
+            <a href="alert.html">
+                <li class="padding-7">Alert</li>
+            </a>
+            <a href="badge.html">
+                <li class="padding-7">Badge</li>
+            </a>
+            <a href="card.html">
+                <li class="padding-7">Card</li>
+            </a>
+            <a href="image.html">
+                <li class="padding-7">Image</li>
+            </a>
+            <a href="input.html">
+                <li class="padding-7">Input</li>
+            </a>
+            <a href="list.html">
+                <li class="padding-7">List</li>
+            </a>
+            <a href="modal.html">
+                <li class="padding-7">Modal</li>
+            </a>
+            <a href="navigation.html">
+                <li class="padding-7">Navigation</li>
+            </a>
+            <a href="grid.html">
+                <li class="padding-7">Grid</li>
+            </a>
+        </div>
```

Comment:
consider following DRY (don't repeat your self principle), either you can populate sidebar with js or you can go with SPA approach which means divide the library in two part i.e

```
<div class="sidebar">
</div>

<div class="main-grid">
   <!-- render content dynamically here on change of event -->
</div>
```

benefits
   - saves loading time
   - gives a single page experience without refreshing 
   - helps to improve readability
   - follows DRY